app/javascript/src/components/Dashboard/Contacts/Form.jsx

```diff
@@ -0,0 +1,94 @@
+import React, { useState } from "react";
+
+import { Formik, Form as FormikForm } from "formik";
+import { Check } from "neetoicons";
+import { Button, Pane, Toastr } from "neetoui";
+import { Input, Select } from "neetoui/formik";
+
+import formInitialValues from "constants/formInitialValues";
+import formValidationSchemas from "constants/formValidationSchemas";
```

Comment:
Not sure why this validation schema is put under another `constants` folder. It should be within the closest scope, which is `./constants.js`
```suggestion
import { VALIDATION_SCHEMA } from "constants";
```

Reply:
Fixed in 9bf2fa8